tests/common.sh

```diff
@@ -97,3 +97,20 @@ if [ ! -e $TESTFILE_15M ]; then
         cat $TESTFILE_150K >> $TESTFILE_15M
     done
 fi
+
+# Adapt file paths passed to fwup on Windows
+# This assumes we're running under Wine, which maps Z:\ to Linux's /
+HOST_FWFILE=$FWFILE
```

Comment:
I'm not 100% sold on this `HOST_` prefix thing because it feels messy. But it does work, so I went with it to help get things moving forward. I'm hoping there's still a better way to have common paths work between Linux and Windows, perhaps using relative instead of absolute paths.